src/command_modules/azure-cli-container/azure/cli/command_modules/container/tests/latest/test_container_commands.py

```diff
@@ -125,7 +125,7 @@ def test_container_create_with_acr(self, resource_group, resource_group_location
         registry_username = 'clitestregistry1'
         registry_server = '{}.azurecr.io'.format(registry_username)
         image = '{}/nginx:latest'.format(registry_server)
-        password = 'UGHH=Ytqp9pMo7IprYJHz9LBKvy0Pwbk'
+        password = 'EsYA8DyuSaNv+3HyGyHoiIIafktf0kE6'
```

Comment:
This is a dummy secret?

Reply:
Yes, not a valid secret. Just a placeholder